website/src/css/customTheme.scss

```
@@ -1103,12 +1103,11 @@ div[class^="tableOfContents"] {
     margin-left: -16px;
     padding-left: 12px;
     color: var(--ifm-font-color-base);
-    border-left: 4px solid var(--brand);
+    border-left: 4px solid var(--ifm-link-color);
 
     code {
-      font-weight: 600;
       color: var(--ifm-font-color-base);
-      white-space: nowrap;
+      font-weight: 900;
```

```suggestion
      font-weight: 700;
```

Black monospace font is bit too much.